pages/public/index.vue

```diff
@@ -7,7 +7,7 @@
       <v-btn class="ma-2" tile outlined color="primary" to="/login">Login</v-btn>
       <v-btn class="ma-2" tile outlined color="primary" to="/dashboard">Dashboard</v-btn>
       <v-btn class="ma-2" tile outlined color="primary" to="/quem_somos">Quem Somos</v-btn>
-      <v-btn class="ma-2" tile outlined color="primary" to="/fale_conosco">Fale Conosco</v-btn>
+      <v-btn class="ma-2" tile outlined color="primary" to="/fale_conosco">Fale Conosco</v-btn>      
```

Comment:
```suggestion
      <v-btn class="ma-2" tile outlined color="primary" to="/fale_conosco">Fale Conosco</v-btn>
```
Sem ajuste, espaço desnecessário